gtsam/nonlinear/NonlinearFactorGraph.h

```diff
@@ -115,6 +115,11 @@ namespace gtsam {
     void saveGraph(std::ostream& stm, const Values& values = Values(),
       const GraphvizFormatting& graphvizFormatting = GraphvizFormatting(),
       const KeyFormatter& keyFormatter = DefaultKeyFormatter) const;
+  
+    /** Write the graph in GraphViz format to file for visualization */
```

Comment:
Mention that this is only here to make wrapping possible, as wrapped languages do not have access to C++ streams.